mirage/config.js

```diff
@@ -254,6 +260,20 @@ export default function configure() {
     return titles.find(request.params.id);
   });
 
+  this.post('/titles', (schema, request) => {
```

Comment:
Do we need to `POST /resources` or `POST /titles`? https://github.com/folio-org/mod-kb-ebsco/pull/109

Reply:
We need to POST /resources. Can only create a custom title (resource) in the context of a custom package

Reply:
I'm a little confused. I thought we were creating that post to RM API from the backend of `/titles`?

Since a new title should post to `/title` on any other API.

Unless this entire PR is meant to be a custom resource and not a custom title?

Reply:
My assumption was that we would create a custom title from the frontend with `POST /titles`. But the backend creates a resource (which creates the title at the same time).

Then when we link additional packages to a title, it creates more resources

Reply:
I think this is the last blocker for this PR.

If we're artificially splitting titles and resources into different routes, I think `mod-kb-ebsco` needs to do that too. Should it not?

The eventual goal is to have this frontend work with other backends. That's going to be infinitesimally harder if we have to do weird things specific to RM API like "create a new title by posting to resources."

Since `mod-kb-ebsco` is supposed to smooth out this weirdness so the frontend can be sane, shouldn't we have a `POST /titles` for creating _titles_?

Again, I know we're actually creating a resource at RM API's level, but if we couple the frontend _that_ tightly to the weirdness of RM API, that is going to be hell for us when it comes to supporting other knowledge bases in the future (if we're able to at all)

Reply:
From the whiteboard session we had a few weeks ago, I think the determination was that `PUT /titles` we could do, but `POST /titles` just isn't really possible in a sane way without RM API changes. Because a title can't be created without a package relationship, the object being created is really a resource (with a new title being created simultaneously).

Reply:
Hence the `packageId` sent with this payload. Couldn't RM API just map `POST /titles` to `POST /resources` behind the scenes?

Besides the `packageId` field, I don't see a difference in POSTing to titles or resource besides one sounds like it should work and the other feels gross.

If we really don't want to have `POST /titles`, I think this should become a new resource page instead of a new title page (`/eholdings/resources/new`).

I'm pretty against having new titles post to resources when we can abstract that away in `mod-kb-ebsco`. We can either call them resources in the frontend too, or add that POST to `/titles`.